src/test/regress/expected/failure_connection_establishment.out

```diff
@@ -81,13 +80,13 @@ SELECT citus.mitmproxy('conn.delay(500)');
 
 -- we cannot control which replica of the reference table will be queried and there is
 -- only one specific client we can control the connection for.
--- by using round-robin task_assignment_policy we can force to hit both machines. We will
--- use two output files to match both orders to verify there is 1 that times out and falls
--- through to read from the other machine
+-- by using round-robin task_assignment_policy we can force to hit both machines. 
```

Comment:
Clarification on the change here:
we talked on this with @thanodnl who has added the original test. The two output file is unnecessary (and not exist in the repo as well), since (a) We made the log level ERROR and ignore WARNINGs (b) via round-robin we enforce that we'll hit the node behind the proxy once. And, output of `dump_network_traffic ` ensures that.

So, the comment seems stale and we update it now.